README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <a href="https://github.com/pyodide/pyodide">
-  <img src="./docs/_static/img/pyodide-logo-readme.png" alt="Pyodide">
+  <img src="https://github.com/pyodide/pyodide/raw/master/docs/_static/img/pyodide-logo-readme.png" alt="Pyodide">
```

Comment:
This is to include it into the sphinx docs as it? It's not great to use Github raw as a CDN and also it would break readme for released tags/branches if that path changed for any reason on master in the future.

Reply:
I think I'm going to get rid of this anyways to put back the separate about page. Unless you think we should include both the `about.md` and `README.md` into the docs?

Reply:
+1 Separate about.md and README.md is easier IMO even if it means keeping some semi-duplicate text in sync.

Reply:
I am going to keep the history and acknowledgements in the README too because I think they fit well in there. So for now about.md is going to just be a subset of the info in the readme. Hopefully we'll fill it out more in the future as you suggested above:

> we could add more thing in the future to about such as a list of current maintainers, other sponsors of there are any etc

Reply:
+1 for keeping history there, particularly given the possible confusion pyodide/iodide, but maybe we could put it after contributing and  before license?

I would rather we removed acknowledgements for the readme though,
 - I put RDT there to be fair, but technically they are already running paid ads on the docs to make it sustainable. As of now we still don't have Mozilla CI support.
 - it would create the expectation that other potential sponsors would also be listed the readme, which I think is not ideal.

Reply:
> I would rather we removed acknowledgements for the readme 

Sounds good. Ideally it would be nice to have an acknowledgments section in the README that acknowledged specific *people* rather than sponsors. But given that we are acknowledging companies I see your point.

We got an acknowledgement of sorts for @mdboom into the README by adding the history section which calls him out as the creator, that seems like the most important one anyways. You would also deserve an acknowledgment but as long as you are the main maintainer it'd be weird. We should probably add a maintainers list, like with in a paper the authors don't acknowledge themselves but they are listed as authors...